lib/kitchen/directions/bake_chapter_references/v1.rb

```diff
@@ -4,28 +4,51 @@ module Kitchen::Directions::BakeChapterReferences
   class V1
     renderable
 
-    def bake(chapter:, metadata_source:, uuid_prefix: '.', klass: 'references')
+    def bake(
+      chapter:,
+      metadata_source:,
+      uuid_prefix: '.',
+      klass: 'references',
+      from_introduction: false
+    )
       @metadata = metadata_source.children_to_keep.copy
       @klass = klass
       @title = I18n.t(:references)
       @uuid_prefix = uuid_prefix
 
-      chapter.references.search('h3').trash
+      chapter.references.search('h3').each(&:trash)
```

Comment:
Did this not work before?  The `trash` method on the enumerator returned by `search` should do exactly this. https://github.com/openstax/kitchen/blob/main/lib/kitchen/element_enumerator_base.rb#L371-L373